web/policies/comment_policy.ex

```diff
@@ -1,8 +1,9 @@
 defmodule CodeCorps.CommentPolicy do
   alias CodeCorps.Comment
   alias CodeCorps.User
+  alias Ecto.Changeset
 
-  def create?(%User{}), do: true
+  def create?(%User{} = user, %Changeset{changes: %{user_id: creator_id}}), do: user.id == creator_id
```

Comment:
I would expect a method name with a `?` to to imply it will return a bool. Thoughts on something like `can_create?`? This would probably be out of the context of this PR, but just a thought.